frontend/src/pages/CalendarPage/components/EventCard/styles.js

```diff
@@ -6,6 +6,16 @@ const styles = theme => ({
     marginBottom: `${theme.spacing(1)}px`,
     margin: `0 ${theme.spacing(0.5)}px ${theme.spacing(1)}px`,
   },
+  modal: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    margin: 'auto 10%',
+  },
+  modalDiv: {
```

Comment:
![image](https://user-images.githubusercontent.com/42155527/118581951-692c3d00-b747-11eb-84f3-3516e2bea4cf.png)
 Changed it to modal-div but got a syntax error, so it will remain as modalDiv.